jkube-kit/build/service/docker/src/main/java/org/eclipse/jkube/kit/build/service/docker/BuildService.java

```diff
@@ -292,7 +298,7 @@ private boolean checkForNocache(ImageConfiguration imageConfig) {
         }
     }
 
-    private boolean isEmpty(String str) {
+    private static boolean isEmpty(String str) {
```

Comment:
Maybe this can be replaced by [StringUtils.isBlank](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isBlank-java.lang.CharSequence-)